lib/origin_simulator/simulation.ex

```diff
@@ -17,6 +17,18 @@ defmodule OriginSimulator.Simulation do
     GenServer.call(server, :recipe)
   end
 
+  def route(server) do
+    GenServer.call(server, :route)
+  end
+
+  # for now, deal with minimum viable recipe: list containing a single recipe
+  def add_recipe(server, new_recipe) when is_list(new_recipe) and length(new_recipe) == 1 do
+    GenServer.call(server, {:add_recipe, new_recipe |> hd})
+  end
+
+  # returning error, pending current work on multi-route / recipes
```

Comment:
A cleaner approach would be to just avoid the comment and add a github comment here (if there is an incoming PR). Comments can so easily go stale or get missed etc. It's this PR yeah https://github.com/bbc/origin_simulator/pull/22